plugins/circleci-npm/readme.md

```diff
@@ -0,0 +1,33 @@
+# `circleci-npm` Tool Kit plugin
+
+This plugin is for managing Tool Kit hooks that are run from npm scripts (such as `npm run test`).
```

Comment:
**nitpick**: this description is copied from the `npm` plugin, but it's not accurate for this plugin